### DISCUSSION TITLE
客座總編輯杜奕瑾 | 人工智慧時代，藝術創作是創造一個張雨生、一個周杰倫（上）

### DISCUSSION THREAD

**Discussion by lukiea** [1101-GS3073-C/Python-Community-GS3073-C#710] [General]
https://futurecity.cw.com.tw/article/365　
是否聽膩了身邊的音樂了，是否覺得歌手更新太慢，ＡＩ　幫你生出你自己最喜歡的音樂，不須再等待

**Comment by Valentinaa0214** [comment]
AI產生的音樂固然有趣，但總覺得缺少了些情感或是溫度，我覺得人類的創作與思考能力還是有其不可取代之處。
不過近期的AI繪圖、AI寫文章機器人陸續出現，能看出AI真的逐漸在創意方面的能力逼近人類了。

**Comment by yjpxi** [comment]
我覺得這種技術的應用不僅能讓已故藝術家的風格得以延續，也能夠讓新的藝術形式誕生，AI能夠在短時間內生成大量的創作樣本，這不僅能激發人類藝術家的靈感，還可能帶來全新的藝術潮流。
